FirebaseVertexAI/Sources/VertexAI.swift

```diff
@@ -149,12 +149,21 @@ public class VertexAI {
 
   private let auth: AuthInterop?
 
-  /// A map of active  `VertexAI` instances keyed by the `FirebaseApp` name and the `location`, in
-  /// the format `appName:location`.
-  private static var instances: [String: VertexAI] = [:]
-
-  /// Lock to manage access to the `instances` array to avoid race conditions.
-  private static var instancesLock: os_unfair_lock = .init()
+  #if compiler(>=6)
+    /// A map of active  `VertexAI` instances keyed by the `FirebaseApp` name and the `location`, in
+    /// the format `appName:location`.
+    private nonisolated(unsafe) static var instances: [String: VertexAI] = [:]
+
+    /// Lock to manage access to the `instances` array to avoid race conditions.
+    private nonisolated(unsafe) static var instancesLock: os_unfair_lock = .init()
+  #else
+    /// A map of active  `VertexAI` instances keyed by the `FirebaseApp` name and the `location`, in
+    /// the format `appName:location`.
+    static var instances: [String: VertexAI] = [:]
```

Comment:
```suggestion
    private static var instances: [String: VertexAI] = [:]
```

Reply:
nit: I think these can remain `private` in the < Swift 6 case.

Reply:
Good catch!